website/content/docs/nia/cli/start.mdx

```diff
@@ -0,0 +1,62 @@
+---
+layout: docs
+page_title: Start Command
+description: >-
+  Consul-Terraform-Sync supports start command for starting CTS as a daemon
+---
+
+# start
+
+The `start` command starts Consul-Terraform-Sync (CTS) as a daemon. When running as a daemon, there is no default configuration to start CTS in a meaningful way. Setting a configuration flag `-config-file` or `-config-dir` is required. For example:
```

Comment:
When CTS runs as a daemon, there is no default configuration to start CTS. You must set a configuration flag -config-file or -config-dir. For example: